src/core/model/workspace.ts

```diff
@@ -1,10 +1,11 @@
-import { Model } from "./model";
-import { ViewSet } from "../view/viewSet";
+import { Model } from './model';
+import { ViewSet } from '../view/viewSet';
+import { Documentation } from '../documentation/documentation';
 
 export abstract class AbstractWorkspace {
     public id!: number;
-    public name: string = "";
-    public description: string = "";
+    public name: string = '';
```

Comment:
I know that many people prefer double over single quotes, but in order to keep consistency, please change back to double quotes. Same goes for some of the `import` statements in this PR.

I am totally open to change from double to single quotes, but only consistently everywhere - and then, not in this PR.

Reply:
Rogue IDE formatting 🤦‍♂